database/migration/src/main/resources/changesets/migration/workflow_store/replace_empty_custom_labels.sql

```diff
@@ -0,0 +1,3 @@
+UPDATE WORKFLOW_STORE_ENTRY
+SET CUSTOM_LABELS = "{}"
+WHERE CUSTOM_LABELS = "";
```

Comment:
I believe it is currently empty. I tested these changes against a 24 db with workflows in flight and things looked good.